styles.css

```diff
@@ -81,8 +88,52 @@ header {
 	text-transform: uppercase;
 }
 
+main {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-evenly;
+}
+
 header img {
 	width: 100%;
-	margin-bottom: -100px;
-	
+}
+
+.portfolio {
+	width: 55%;
+	height: 1000px;
+	padding: 20px;
+	overflow: scroll;
+}
+
+.portfolio-card {
+	border: 1px solid;
+	padding: 20px;
+	margin: 20px 0px;
+}
+
+.portfolio-card img{
+	width: 40%;
+}
+
+.about {
```

Comment:
Might be nice to add a border around this section to make it to distinguish it from the projects section